### PR TITLE
Move profiler docs to L1 in the nav

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Moves Continuous Profiler out from underneath APM
- Update home page
- move files in directory structure and add aliases
- move images to match
- update links and img refs
- update partials
- update nav category name

### Motivation
Discussion with APM and Profiler teams

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: data/partials/home.yaml [hunk 1]
```diff
@@ -48,10 +48,14 @@ nav_sections:
...
         desc: Create, edit, and manage your monitors and notifications
-      - title: APM & Continuous Profiler
+      - title: APM
         link: tracing/
         icon: apm
         desc: Explore out-of-the-box performance dashboards and distributed traces
+      - title: Continuous Profiler
+        link: profiler/
+        icon: profiling-1
+        desc: See and compare performance snapshots and find problem code
```
Comment (alai97) [inline data/partials/home.yaml:58]:
```suggestion
        desc: Compare performance snapshots and investigate bottlenecks in traces
```

File: content/en/profiler/_index.md [hunk 2]
```diff
@@ -2,15 +2,16 @@
...
+    - link: '/profiler/enabling'
       tag: 'Documentation'
       text: 'Enable continuous profiler for your application.'
     - link: 'getting_started/profiler'
       tag: 'Documentation'
       text: 'Getting Started with Continuous Profiler'
-    - link: 'tracing/profiler/search_profiles'
+    - link: 'profiler/search_profiles'
       tag: 'Documentation'
       text: 'Learn more about available profile types.'
```
Comment (alai97) [inline content/en/profiler/_index.md:16]: There's also one more link that has a period in Line 28 (Compare and optimize).

```suggestion
      text: 'Learn more about available profile types'
```

File: content/en/profiler/_index.md [hunk 3]
```diff
@@ -2,15 +2,16 @@
...
 kind: Documentation
 aliases:
-    - /tracing/profiling
+    - /tracing/profiling/
+    - /tracing/profiler/
 further_reading:
-    - link: '/tracing/profiler/enabling'
+    - link: '/profiler/enabling'
       tag: 'Documentation'
       text: 'Enable continuous profiler for your application.'
```
Comment (alai97) [inline content/en/profiler/_index.md:10]:
```suggestion
      text: 'Enable continuous profiler for your application'
```

File: content/en/profiler/connect_traces_and_profiles.md [hunk 4]
```diff
@@ -5,12 +5,14 @@ further_reading:
     - link: 'tracing'
       tag: 'Documentation'
       text: 'APM Distributed Tracing'
-    - link: '/tracing/profiler/enabling'
+    - link: '/profiler/enabling'
       tag: 'Documentation'
       text: 'Enable Continuous Profiler for Your Application.'
```
Comment (alai97) [inline content/en/profiler/connect_traces_and_profiles.md:10]:
```suggestion
      text: 'Enable Continuous Profiler for Your Application'
```

File: content/en/profiler/compare_profiles.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 title: Compare Profiles
 kind: documentation
 further_reading:
-    - link: 'tracing/profiler/enabling'
+    - link: 'profiler/enabling'
       tag: 'Documentation'
       text: 'Enable continuous profiler for your application'
     - link: 'getting_started/profiler'
```
Comment (alai97) [inline content/en/profiler/compare_profiles.md:8]: The preview link is not generating for these two Blog links due to indentation issues.

Comment (alai97) [reply content/en/profiler/compare_profiles.md:8]: Also please remove the period after "Comparison" in Line 16.

**Review by alai97 — APPROVED** [review]

File: data/partials/home.yaml (hunk 1 above)
Comment (kayayarai) [reply data/partials/home.yaml:58]:
```suggestion
        desc: Compare performance snapshots and investigate bottlenecks
```

File: content/en/profiler/compare_profiles.md (hunk 5 above)
Comment (kayayarai) [reply content/en/profiler/compare_profiles.md:8]: Nice catch, it's actually the use of `tags:` instead of `tag:` that is causing that. Fixed

**Pull request merged**